chapter06/pxd/ex3.pl

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use feature "say";
+
+
+my $col_width = 20;
+my %env = %ENV;
```

Comment:
I just got a copy of the %ENV in case the actual vars change during execution of the script (highly unlikely! just cautious)